yoga/Yoga.c

```diff
@@ -2197,6 +2197,7 @@ static void YGNodelayoutImpl(const YGNodeRef node,
     // If the main dimension size isn't known, it is computed based on
     // the line length, so there's no more space left to distribute.
 
+    bool sizeBasedOnContent = false;
```

Comment:
Or just remove the boolean all together and inline `measureModeMainDim == YGMeasureModeExactly`